<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Jason R. Dobson's Home Page">
    <meta name="author" content="Jason R. Dobson">
    <link rel="shortcut icon" href="image_assets/favicon.png">

	<title>Jason R. Dobson's Home Page</title>

    <!-- Bootstrap core CSS -->
    <link rel="stylesheet" href="dist/css/bootstrap.css"/>

    <!-- Custom styles for this template -->
    <link href="blog.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>
<body>

    <div class="blog-masthead">
      <div class="container">
        <nav class="blog-nav">
          <a class="blog-nav-item active" href="#">Home</a>
          <a class="blog-nav-item active" href="http://www.keystonesymposia.org/14Q2">Keystone Meeting Page</a>
        </nav>
      </div>
    </div>

    <div class="container">

      <div class="blog-header">
        <h1 class="blog-title">Keystone Symposia on Molecular and Cellular
            Biology</h1>
        <p class="lead blog-description">Transcriptional Regulation Q2 2014</p>
      </div>

      <div class="row">

        <div class="col-sm-8 blog-main">

          <div class="blog-post">
            <a name="Bradner"></a>
            <h2 class="blog-post-title">James Bradner</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Bradner Q1: 0101</h1>

            <h3>Chemical modulation of chromatin structure and function</h3>

            <p>Describes cancer as a disease of human enhancers</p>

            <p>c-MYC preferentially occupies superenhancers</p>

            <ul>
            <li>invades new enhancers</li>
            <li>recruits elongation factors</li>
            </ul>

            <p>One such factor is BRD4, which can be inhibited by their novel small molecule
            JQ1</p>

            <p>Superenhancers are not only read at highly expressed genes but at all genes</p>

            <p>JQ1 inhibits pol II progression through gene bodies</p>

            <h3>Gave a TED talk - worth looking up</h3>

            <h3>Brander's view is to do <em>in vivo</em> expts. first to get figs 1 &amp; 5</h3>
          </div>

          <div class="blog-post">
            <a name="Thompson"></a>
            <h2 class="blog-post-title">Craig Thompson</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Thompson Q1: 0102</h1>

            <h3>Link between metabolism (glucose/glutamine) and histone acetylation</h3>

            <p>Studying IDH1 mutations in chromosomally normal AML</p>

            <p>Phenotypic genes in AML are in heterochromatin [PcG/HP1]</p>

            <p>nsSNV (missense) mutations in IDH1/2 are clustered in the active site</p>

            <p>IDH1/2 mutations are mutually exclusive with TET2</p>

            <p>2HG (rare metabolite) accumulation with mutant IDH &amp; inhibits TET2 &amp; jumonji
            domain members</p>
          </div>
          <div class="blog-post">
            <a name="Conaway"></a>
            <h2 class="blog-post-title">Joan Conaway</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Conaway Q1: 0103</h1>

            <p>Did not follow this talk at all.</p>

            <p>Had some really pretty fluorescent westerns.</p>
          </div>
          <div class="blog-post">
            <a name="Cramer"></a>
            <h2 class="blog-post-title">Patrick Cramer</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Cramer Q1: 0104</h1>

            <h3>Transcriptome surveillance by selective termination of noncoding RNA synthesis</h3>

            <h4>Unpublished data</h4>

            <h4>All yeast data</h4>

            <p>Used PAR-clip to understand mRNA biogenesis</p>

            <ul>
            <li>Zero-length crosslinking that uses UV</li>
            </ul>

            <p>Splicing factors at the 3'-end of genes appear to be critical for poly-A</p>
          </div>
          <div class="blog-post">
            <a name="Ren"></a>
            <h2 class="blog-post-title">Bing Ren</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Ren Q1: 0105</h1>

            <h3>Long-range control sequences in health and disease</h3>

            <h4>TADs don't change during differentiation => rather they switch compartments</h4>

            <h5>Moderate topological changes are linked with enhancers marked by H3K4me1</h5>

            <h4>Used TALENs to knock in modified OCT4 to knock out enhancer sites</h4>

            <h3>PUSHED RESOLUTION OF HiC TO RES ENZ LEVEL USING 3.4 BILLION PE READS AND NEW COMPUTATIONAL METHOD</h3>

            <ul>
            <li>5-10Kb fragment length - asked question about using sonication instead of
            ResEnz - did not try didn't seem enthused</li>
            </ul>

            <p>Jin et al Nature [in press]</p>

            <p>Haplo-seq [NBT] // Dixon has a new paper coming out</p>
          </div>
          <div class="blog-post">
            <a name="Roeder"></a>
            <h2 class="blog-post-title">Robert Roeder</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Roeder Q2: 0106</h1>

            <h3>Looking at linker histones to model chromatin structure ["30nm" vs "10nm"]</h3>

            <p><em>in vitro</em> chromatin assembly</p>

            <p>Did H1-Flag IP->MassSpec
            * Found NAP1 which can remove H1 30nm->10nm through acetylation of chroamtin
              [not clear which residues]</p>
          </div>
          <div class="blog-post">
            <a name="Bulyk"></a>
            <h2 class="blog-post-title">Martha Bulyk</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Bulyk Q2: 0107</h1>

            <h3>Transcription factors and DNA regulatory elements</h3>

            <p>Showed permutations of NFkB homo/heterodimers and DNA binding elements</p>

            <p>Really hard to distinguish one motif from another - not too different</p>

            <p>Some confusion over motif lengths</p>
          </div>
          <div class="blog-post">
            <a name="Nogales"></a>
            <h2 class="blog-post-title">Eva Nogales</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Nogales Q2: 0108</h1>

            <h3>Structure of the human transcription preinitiation complex</h3>

            <h4>Single particle EM</h4>

            <ul>
            <li>2D images</li>
            <li>image processing</li>
            <li>Reconstruct 3D images
            <h3>NOT Z STACKS</h3></li>
            </ul>

            <p>Rest of talk was a bunch of pictures of factor reassembly did not get the take
            home message</p>
          </div>
          <div class="blog-post">
            <a name="Price"></a>
            <h2 class="blog-post-title">David Price</h2>
            <p class="blog-post-meta">February 5, 2014</p>

            <h1>Price Q2: 0109</h1>

            <h3>Functional domains of Gdown1 and regulation by phosphorylation</h3>

            <p>Gdown1 enhaces proximal paused Pol II</p>

            <p>Increased NELF as well as K4me1</p>

            <p>Phos. of Gdown1 releases from pausing => critical for mitosis</p>
          </div>
          <div class="blog-post">
            <a name="Levine"></a>
            <h2 class="blog-post-title">Michael Levine</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Levine Q2: 0201</h1>

            <p>Early embryonic development</p>

            <p>Paused polymerase - gro-seq</p>

            <p>Snail regulates invagination of mesoderm</p>

            <p>Synchronous expression of genes are controlled by level of paused Pol II read counts.</p>

            <p>Used sna enhancer and minimal promoter elements</p>

            <ul>
            <li>Promoter            Pol II Read     Sync expression</li>
            <li>SNAIL promoter   100             high</li>
            <li>sog              50              med</li>
            <li>ths              15              low</li>
            </ul>

            <p>Showed a video of synchronous cell division - onset of gastrulation - flickering bursts of transcription - Weakly paused Thisbe promoter</p>

            <ul>
            <li>Average burst ~2-5 min, ~10-50 trans./burst</li>
            </ul>

            <p>Strongly paused promoter (Snail) - "less bursty"</p>

            <ul>
            <li>More stable single - must less flickering</li>
            </ul>

            <h3>Model: Paused Pol II Suppresses Bursting</h3>

            <p>Showed beautiful video of broad expression of eve stripe 2 being attenuated into a well-defined stripe at onset of gastrulation</p>

            <h3>Model: Bursts foster dynamic repression of gene expression</h3>

            <p>Used MSL2 stem loop structure to control pausing</p>

            <ul>
            <li>Need more info on this - check paper</li>
            </ul>

            <p>Appears to be a disconnect between observations and proposed mechanism</p>
            
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Adelman"></a>
            <h2 class="blog-post-title">Karen Adelman</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Adelman Q2: 0202</h1>

            <h1>RNA Polymerase II Pausing in Pluripotent Stem Cells</h1>

            <h3>Unpublished data</h3>

            <h2>NELF and P-TEFb are key regulators of pause duration</h2>

            <h3>Work from John Lis</h3>

            <ul>
            <li>NELF inhibits early txn elongation</li>
            <li>P-TEFb kin stim. diss. of NELF</li>
            <li>NELF is "stupid" and goes everywhere with P2</li>
            <li>P-TEFb can be modulated and controlled to release pausing</li>
            </ul>

            <h2>Most dros genes that have Pol II has promoter enrichment, which suggests there is pausing at most genes</h2>

            <h2>Pausing index = promoter density / downstream density</h2>

            <h2>Pausing is not an on/off switch - rather most of these genes are on at some level - leaky - not a good way to turn on/off more for attenuation</h2>

            <h2>Genes paused were enriched in developmental genes (S2 cells)</h2>

            <h2>Question: how to define where NELF-mediated pausing is functionally important?</h2>

            <h3>Pausing plays an important role in preventing nucleosomes from forming - absence of NELF => increased nucleosome formation</h3>

            <h2>Dual functions of pausing</h2>

            <ul>
            <li>Limits P2 release</li>
            <li>Prevents promoter nucleosome assembly</li>
            </ul>

            <h2>What is the role in mammalian development?</h2>

            <h2>Bivalent and dev. genes are not enriched with PP2</h2>

            <h3>Some genes have it but not different than genomic average</h3>

            <h3>What genes are pausing - what is unique about them?</h3>

            <h3>Signaling cascades that regulate development</h3>

            <h3>Some downstream targets had pausing - but not necessary or required for rapid activation</h3>

            <h2>Generated conditional NELF-B KO mouse</h2>

            <h3>Null is lethal - NELF is essential - PI lethality</h3>

            <h3>Moved to tamoxifen CRE</h3>

            <h3>Loss of NELF-B results in degradation of the other subunits</h3>

            <h3>Did microarray immediately after NELF was depleted - early target identification 1000 genes</h3>

            <h4>Genes involved in lineage com. and pluri. not affected</h4>

            <h4>Mapk/Erk signaling was affected</h4>

            <h4>Erk/Mapk signaling activates exit from true naive pluri state</h4>

            <h4>2i media inhibits Map/Erk to maintain a more naive state</h4>

            <h4>Two effects</h4>

            <ul>
            <li>Increased pol2 processing into gene body  =>  upregulation</li>
            <li>lack of Pol II at promoter                =>  downregulation
            <h4>Cells fail to differentiate as indicated by the endoderm marker GATA4</h4></li>
            </ul>

            <h4>FGF doesn't phos ERK - broad dampening of pathway</h4>

            <h4>Can rescue with a fusion of Erk2Mek1 with NLS => get GATA4 expression +/- NELF-b</h4>
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Wysocka"></a>
            <h2 class="blog-post-title">Joanna Wysocka</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Wysocka Q@: 0203</h1>

            <h1>Enhancers are most dynamically utilized part of the mammalian genomes</h1>

            <h2>Looking at transition between two closely related states</h2>

            <ul>
            <li>Preimplanation epiblast - ESC</li>
            <li>postimplantation epiblast - EpiSC
            FGF is major driving signal
            <h2>Came up with <em>in vitro</em> method of making EpiLC - unstable but more similar to <em>in vivo</em> state</h2></li>
            </ul>

            <h3>Used model to investigate differences in enchacers in naive and primed cells</h3>

            <h2>Oct4 undergoes major reorganization of occupancy ESC->EpiLC</h2>

            <h2>Enahcers (p300, k27ac, k4me1) also show cell-type specific reorganization</h2>

            <h3>Same Oct4 motif is recognized by ChIP and ChIP-exo in both cell types - no change in binding specificity</h3>

            <h3>TxN factor cooperativity/compeition => found differentially enriched motifs and MassSpec found similar set of candidates</h3>

            <h3>Changes in candidates correlated very highly with Oct4 changes => especially at enhancers</h3>

            <h3>Differentially occupied sites have changes in nucleosome occupancy by both DNAse and FAIRE-seq</h3>
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Fuller"></a>
            <h2 class="blog-post-title">Margaret Fuller</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Fuller Q2: 0204</h1>

            <h1>Cell type specific transcriptional repression programs proper differentiation in an adult stem cell lineage</h1>

            <h2>Adult stem cells responsible for regeneration of multiple tissue types</h2>

            <h2>Spermatogenesis in drosophila is a model of adult stem cell lineage</h2>

            <h3>Asymetric division -> proliferation -> spermatocytes -> terminal differentiation</h3>

            <h4>Special G2 stage where >1000 genes are turned on (pre-meitoic)</h4>

            <h4>tTAF   =>  Testis-specific components of TFIID are expressed</h4>

            <h4>tMAC   =>  spermatocyte specific form of (MIP/DREAM) complex</h4>

            <h5>Mechanism =></h5>

            <ul>
            <li>Precursors have:
            <ul>
            <li>K27me3</li>
            <li>no k4me3</li>
            <li>no p2</li>
            </ul></li>
            <li>Then genes are switched:
            <ul>
            <li>Get pol II</li>
            <li>paused</li>
            </ul></li>
            <li>Pol II gets activated at terminal differentiation</li>
            </ul>

            <h2>Switch from pre-cursors to terminal diff is both activation of proper lineage genes and concomittant repression of other diff. program genes</h2>

            <h3>tZnF is a 6 C2H2 znf only expressed in testis</h3>

            <h4>KO results in meiotic arrest and absence of terminal differentiation</h4>

            <h4>Functions as a repressor - KO has activation of many genes that should never be on in testis - mostly neural genes</h4>

            <h2>Hypothesis: tZnF represses somatic gene expression to prevent collateral damage gene expression upon loss of MMB (DREAM) function in differentiating cells</h2>

            <h3>Showed that somatic genes require tZnF to be repressed in testis (normally repressed by DREAM) and require tMAC for this repressive function</h3>

            <h4>Also showed at transcriptome level</h4>

            <h2>Question: what is the motif?</h2>

            <h3>A: not known => hypothesize that it acts like REST (acts in non-neuronal tissues to repress neuronal genes)</h3>
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Chang"></a>
            <h2 class="blog-post-title">Howard Chang</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Chang</h1>

            <h1>lncRNAs and enhancers</h1>

            <h1>Relationships between structure and function for RNA molecules</h1>

            <h2>HOTTIP controls activation of locus through 3D-chromosome structure</h2>

            <h3>Suggest that HOTTIP provides interface between chromatin state and architecture</h3>

            <h1>NEST => lncRNA confers susceptibility to infection</h1>

            <h2>Interacts with MLLs to promote H3K4me3 and activation of IFN-g expression</h2>

            <h1>WDR5: adapter protein for multiple inputs (RpBP5, MLL1, Dppy30, Ash2L, NEST)</h1>

            <h2>Used crystal structure to mutate all charged residues to alanine (systematic mutational analysis)</h2>

            <h3>Found RNA-binding pocket in WDR5</h3>

            <h4>Found one mutant that only affects RNA-binding => not assembly of the rest of the complex</h4>

            <h4>This mutation can not properly activate target genes</h4>

            <h3>Made an inducible replacement system in mouse (can replace WT WDR5 w/mutatnt in a temporally-defined manner)</h3>

            <h4>ESCs are unable to maintain H3K4me3</h4>

            <h3>Did a screen (tandem-IP contruct) to identify other RNAs that can bind to WT WDR5 and not MT-WDR5</h3>

            <h4>223 ncRNA, 1181 mRNAs, 17 miRNA, 13 snoRNA</h4>

            <h4>Some of these RNAs are important for maintenance of ESC pluripotency</h4>

            <h1>Going in the other direction => mapping RNA structure to function</h1>

            <h2>Parallel Analysis of RNA Structure (PARS)</h2>

            <h3>RNAse V1 library => dsRNA cutting   =>   deep sequencing</h3>

            <h3>RNAse S1 library => ssRNA cutting   =>   deep sequencing</h3>

            <h3>Provides a single nucleotide resolution map of RNA structure</h3>

            <h4>secondary structure of mRNAs</h4>

            <h5>5' CDS loop</h5>

            <h5>3 nucleotide periodicity within CDS</h5>

            <h3>Genomic crossing to look at SNVs effects on RNA shape/structure (maternal/paternal contributions) [RiboSNitch]</h3>

            <h3>RNA structure is different at splice cites</h3>

            <h3>RNA structure is different at true miRNA target sites</h3>

            <h3>RiboSNitches obey systematic rules</h3>

            <h4>Base composition and location at center versus boundary of RSS matter</h4>

            <h2>Can evolution allow us to understand which structural variants are more important and therefore "functional"?</h2>

            <h3>Yes => 3'-UTR, miRNA, and known sites for RNA-binding proteins (not so much lncRNA sequences)</h3>

            <h1>Recently developed a chemical to do this in cell lines (hESCs)</h1>
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Rinn"></a>
            <h2 class="blog-post-title">John Rinn</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Rinn</h1>

            <h1>Single cell RNA-sequencing during differentiation</h1>

            <h5>Fluidics devide that traps a single cell</h5>

            <h5>Problem of abundance</h5>

            <h2>Used myogenesis as a model system</h2>

            <h3>Captured ~100 cells per timepoint in the fluid device => use a microscope to count # of cells</h3>

            <h4>NextEra tech to make library right in device</h4>

            <h2>Proposes that many cells are "out of order" "Monocole"</h2>

            <ol>
            <li>Orders cells by progress though a biological process</li>
            <li>Classifies cells according to type</li>
            <li>Finds differentially expressed genes (unsupervised)
            <h2>Iterative component analysis</h2></li>
            </ol>

            <h3>Reduce dimensionality</h3>

            <h3>Minimal spanning tree - lowest number of lines to connect dots</h3>
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Glass"></a>
            <h2 class="blog-post-title">Christopher Glass</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Glass</h1>

            <h1>Enhancer RNAs; Noise, Consequqnce or Contributors to Enhancer Function?</h1>
          </div><!-- /.blog-post -->
          <div class="blog-post">
            <a name="Barski"></a>
            <h2 class="blog-post-title">Artem Barski</h2>
            <p class="blog-post-meta">February 6, 2014</p>

            <h1>Barksi : not sure which abstract</h1>

            <h1>Something about T cell differentiation</h1>

            <h2>Same immune signaling results in different gene expression programs in naive and memory t-cells</h2>

            <h3>"Silent" genes that are responsive to signaling had H3K4me3 => called these poised</h3>
          </div><!-- /.blog-post -->

        </div><!-- /.blog-main -->

        <div class="col-sm-3 col-sm-offset-1 blog-sidebar">
          <div class="sidebar-module sidebar-module-inset">
            <h4>About</h4>
            <p>This page contains various notes from the Keystone Symposia on
            Molecular Biology and Cell Biology meeting on Transcriptional
            Regulation</p>
          </div>
          <div class="sidebar-module">
            <h4>Speakers</h4>
            <ol class="list-unstyled">
              <li><a href="#Bradner">James Bradner</a></li>
              <li><a href="#Thompson">Craig Thompson</a></li>
              <li><a href="#Conaway">Joan Conaway</a></li>
              <li><a href="#Cramer">Patrick Cramer</a></li>
              <li><a href="#Ren">Bing Ren</a></li>
              <li><a href="#Roeder">Robert Roeder</a></li>
              <li><a href="#Bulyk">Martha Bulyk</a></li>
              <li><a href="#Nogales">Eva Nogales</a></li>
              <li><a href="#Price">David Price</a></li>
              <li><a href="#Levine">Michael Levine</a></li>
              <li><a href="#Adelman">Karen Adelman</a></li>
              <li><a href="#Wysocka">Joanna Wysocka</a></li>
              <li><a href="#Fuller">Margaret Fuller</a></li>
              <li><a href="#Chang">Howard Chang</a></li>
              <li><a href="#Rinn">John Rinn</a></li>
              <li><a href="#Glass">Christopher Glass</a></li>
              <li><a href="#Barski">Artem Barski</a></li>
            </ol>
          </div>
        </div><!-- /.blog-sidebar -->

      </div><!-- /.row -->

    </div><!-- /.container -->

    <div class="blog-footer">
      <p>Blog template built for <a href="http://getbootstrap.com">Bootstrap</a> by <a href="https://twitter.com/mdo">@mdo</a>.</p>
      <p>
        <a href="#">Back to top</a>
      </p>
    </div>


    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
    <script src="../../dist/js/bootstrap.min.js"></script>
    <script src="../../assets/js/docs.min.js"></script>
  </body>
</html>
