<h1>Dekker</h1>

<h3>Topologically associated domains: cell type and cell cycle specific interactions</h3>

<h4>Organization of 3D genome controls gene regulatory dynamics</h4>

<h4>Interphase chromosomes are compartmentalized (active/inactive)</h4>

<ol>
<li>Compartment A</li>
<li>Compartment B</li>
</ol>

<h4>Gene regulation happens within these defined topological domains</h4>

<h5>Used 5C to model this because you only need about 10mil read to get to restriction enzyme resolution for 5Mb region</h5>

<h4>TAD domain boundaries are consistent between cell lines</h4>

<h4>Internally (within TAD boundaries) there can be reorganization within TADs</h4>

<h5>Show cell-type specific enhancer-promoter interactions</h5>

<h3>Genes are spatially restricted to their TAD</h3>

<h3>TADs are universal structural building blocks within wich cell-type specific looping is constrained</h3>

<h3>Reccomends paper: Symmons et al Gen. Res 2014</h3>

<h3>Cell cycle dependent chromosome conformation?</h3>

<h4>During mitosis the interaction map is "boring" - very uniform map</h4>

<h3>Mitotic chromosomes: no compartments</h3>

<h4>Mitotic chromosome folding:</h4>

<ol>
<li>no locus-specific structures</li>
<li>long-range interactions up to 10Mb</li>
</ol>

<h4>Two models:</h4>

<ol>
<li>polymer model</li>
<li>laemmeli scaffold model</li>
</ol>

<h4>Testing polymer model of mitotic chromosomes</h4>

<ol>
<li>polymer (hierarchical model) was not consistent with chromatin interaction map</li>
<li>Laemmeli array loop model reproduce chromatin interaction map (loops 80-120Kb)</li>
</ol>

<h4>Stochastic loop positions are necessary - can't be fixed - each cell in population has different anchors</h4>
