<h1>Chedin</h1>

<h3>Transcription protects CpG island promoters from DNA methylation independently of Histone H3 lysine 4 trimethylation</h3>

<h3>CpG island promoters: islands of protection</h3>

<p>Rarely methylated - even though they appear biochemically to be strong targets/substrates</p>

<h4>Presence of me on H3K4 prevents DNMTs from binding and methylating DNA</h4>

<h3>R-loops</h3>

<ul>
<li><p>3-stranded structure with an RNA:DNA hybrid and a single DNA strand</p></li>
<li><p>Co-transcriptional</p></li>
<li><p>Can be long 0.5-1Kb</p></li>
<li><p>Very stable</p></li>
<li><p>Can be predicted by GC-skew</p></li>
</ul>

<h4>Human CpG islands have a strong GC-skew</h4>

<p>Especially at TSS - Ginno et al Gen Res 2013</p>

<h5>Conserved in vertebrates - not other metazoa unless organism has homolog of DNMT and have global DNA methylation</h5>

<h4>DRIP can recognize and map R-loops</h4>

<h4>R-loop formation is very prevelant at CGI promoters</h4>

<h4>CFP1 has a CXXC that binds CpG island DNA and in a complex with SET1</h4>

<h5>CFP1 -/- show sharply reduced promoter H3K4me3</h5>

<h5>Somehow gene expression isn't changed [odd]</h5>

<h5>Genome was hypomethylated</h5>

<h5>But CGI promoters still protected</h5>

<h4>Inhibit TxN => R-loops disappear rapidly => mut. cells gain high CpGme</h4>

<h4>Genes that are re-methylated were highly expressed</h4>
