<h1>Chang</h1>

<h1>lncRNAs and enhancers</h1>

<h1>Relationships between structure and function for RNA molecules</h1>

<h2>HOTTIP controls activation of locus through 3D-chromosome structure</h2>

<h3>Suggest that HOTTIP provides interface between chromatin state and architecture</h3>

<h1>NEST => lncRNA confers susceptibility to infection</h1>

<h2>Interacts with MLLs to promote H3K4me3 and activation of IFN-g expression</h2>

<h1>WDR5: adapter protein for multiple inputs (RpBP5, MLL1, Dppy30, Ash2L, NEST)</h1>

<h2>Used crystal structure to mutate all charged residues to alanine (systematic mutational analysis)</h2>

<h3>Found RNA-binding pocket in WDR5</h3>

<h4>Found one mutant that only affects RNA-binding => not assembly of the rest of the complex</h4>

<h4>This mutation can not properly activate target genes</h4>

<h3>Made an inducible replacement system in mouse (can replace WT WDR5 w/mutatnt in a temporally-defined manner)</h3>

<h4>ESCs are unable to maintain H3K4me3</h4>

<h3>Did a screen (tandem-IP contruct) to identify other RNAs that can bind to WT WDR5 and not MT-WDR5</h3>

<h4>223 ncRNA, 1181 mRNAs, 17 miRNA, 13 snoRNA</h4>

<h4>Some of these RNAs are important for maintenance of ESC pluripotency</h4>

<h1>Going in the other direction => mapping RNA structure to function</h1>

<h2>Parallel Analysis of RNA Structure (PARS)</h2>

<h3>RNAse V1 library => dsRNA cutting   =>   deep sequencing</h3>

<h3>RNAse S1 library => ssRNA cutting   =>   deep sequencing</h3>

<h3>Provides a single nucleotide resolution map of RNA structure</h3>

<h4>secondary structure of mRNAs</h4>

<h5>5' CDS loop</h5>

<h5>3 nucleotide periodicity within CDS</h5>

<h3>Genomic crossing to look at SNVs effects on RNA shape/structure (maternal/paternal contributions) [RiboSNitch]</h3>

<h3>RNA structure is different at splice cites</h3>

<h3>RNA structure is different at true miRNA target sites</h3>

<h3>RiboSNitches obey systematic rules</h3>

<h4>Base composition and location at center versus boundary of RSS matter</h4>

<h2>Can evolution allow us to understand which structural variants are more important and therefore "functional"?</h2>

<h3>Yes => 3'-UTR, miRNA, and known sites for RNA-binding proteins (not so much lncRNA sequences)</h3>

<h1>Recently developed a chemical to do this in cell lines (hESCs)</h1>
